clients/cloud/python/requirements.txt

```diff
@@ -0,0 +1,7 @@
+requests
+fastavro>=0.23.0
+avro-python3==1.9.2.1
+jsonschema
+protobuf
+confluent-kafka
```

Comment:
Since this isn't a version-locked environment file (nor should it be for this purpose), my temptation is to replace this entire file with the single entry:

```
confluent-kafka[avro,jsonschema,protobuf]
```
This installs the lib and the extras for schema variants, plus all the transitive dependencies using `confluent-kafka`'s `setup.py`.  It avoids mentioning transitive dependencies and versions which might in-future become non-dependencies or have different bounds.

If favoured, I can send a PR.